src/EditorFeatures/CSharpTest/Diagnostics/UpgradeProject/UpgradeProjectTests.cs

```diff
@@ -531,18 +531,19 @@ void A()
         [Fact]
         public async Task OnlyOfferFixAllProjectsFromCSharp6ToDefaultWhenApplicable()
         {
+            var defaultVersion = LanguageVersion.Default.MapSpecifiedToEffectiveVersion().ToDisplayString();
```

Comment:
Nitpick: I'd call this effectiveVersionOfDefaultVersion. Otherwise the asserts below make it look like we're asserting it *does* say "default" which is not the case.